### PR TITLE
feat(email-first): Email-first everywhere!

### LINKED ISSUE #3204: Use email-first as default for "web" broker
The "web" broker is for users that sign into FxA when browsing directly to https://accounts.firefox.com, and is used a lot in our functional tests.

blocks #639

### PR DESCRIPTION
**Pull request by shane-tomlinson**
fixes #3204

### CODE REVIEW

File: packages/fxa-content-server/tests/functional.js [hunk 1]
```diff
@@ -22,7 +22,6 @@ module.exports = [
   'tests/functional/cookies_disabled.js',
   'tests/functional/delete_account.js',
   'tests/functional/email_opt_in.js',
-  'tests/functional/email_service.js',
```
Comment (vladikoff) [inline packages/fxa-content-server/tests/functional.js]: @shane-tomlinson did you want to delete this? this is unrelated to email_first ?

Comment (shane-tomlinson) [reply packages/fxa-content-server/tests/functional.js]: It is unrelated, but we are using the email-service everywhere now so it seems like a bit of a superfluous test.

File: packages/fxa-content-server/app/scripts/views/sign_in_unblock.js [hunk 2]
```diff
@@ -52,16 +52,17 @@ const View = FormView.extend({
     );
   },
 
-  onSignInError(account, password, err) {
-    if (AuthErrors.is(err, 'INCORRECT_PASSWORD')) {
+  onSignInError(account, password, error) {
+    if (AuthErrors.is(error, 'INCORRECT_PASSWORD')) {
       // The user must go enter the correct password this time.
       this.navigate(this._getAuthPage(), {
+        account: this.getAccount(),
```
Comment (shane-tomlinson) [inline packages/fxa-content-server/app/scripts/views/sign_in_unblock.js]: This was needed to get signin unblock to go to the /signin page where the user could enter their password rather than to `/`

File: packages/fxa-content-server/tests/functional.js (hunk 1 above)
Comment (shane-tomlinson) [reply packages/fxa-content-server/tests/functional.js]: Brought the test back.

**Comment by shane-tomlinson** [comment]
I'm calling this ready for review, @mozilla/fxa-devs - r?

The main change here is to make the "web" broker (the fallback when user browses directly to the site) use email-first. That caused a *ton* of test upheaval.

File: packages/fxa-content-server/tests/functional/change_password.js [hunk 3]
```diff
@@ -57,9 +57,6 @@ registerSuite('change_password', {
     email = TestHelpers.createEmail();
   },
 
-  afterEach: function() {
```
Comment (vbudhram) [inline packages/fxa-content-server/tests/functional/change_password.js]: It isn't obvious to me why this was removed?

Comment (shane-tomlinson) [reply packages/fxa-content-server/tests/functional/change_password.js]: Because we do it in the beforeEach, and I think everywhere should do it in beforeEach. Doing it in afterEach and then beforeEach in the subsequent tests just slows things down.
